projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid-navigation.service.ts

```diff
@@ -1,12 +1,53 @@
 import { IgxGridNavigationService } from '../grid-navigation.service';
 import { Injectable } from '@angular/core';
 import { IgxPivotGridComponent } from './pivot-grid.component';
+import { HEADER_KEYS } from '../../core/utils';
 
 @Injectable()
 export class IgxPivotGridNavigationService extends IgxGridNavigationService {
     public grid: IgxPivotGridComponent;
 
-    public dispatchEvent(event: KeyboardEvent) {
-        // TODO
+    public isRowHeaderActive: boolean;
+
+    public get lastRowDimensionsIndex() {
+        return this.grid.rowDimensions.length - 1;
+    }
+
+    public focusOutRowHeader() {
```

Comment:
This seems to be called only on pointer down of certain elements, but not in all cases when your focus moves outside the pivot row header elements. If you click for example on the column header this does not get called so isRowHeaderActive remains true.